### PR TITLE
CircleCI: Increase nodejs max memory

### PR DESCRIPTION
**Pull request by dprokop**
As of https://support.circleci.com/hc/en-us/articles/360009208393-How-can-I-increase-the-max-memory-for-Node-

Not sure this will help, but let's see

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -1168,6 +1172,10 @@ jobs:
     working_directory: /go/src/github.com/grafana/grafana
     steps:
       - checkout
+      - run:
+        name: Increase node momory
+        command: |
+          export NODE_OPTIONS=--max_old_space_size=4096
```
Comment (aknuds1) [inline .circleci/config.yml]: You need indentation here.

**Review by aknuds1 — CHANGES_REQUESTED** [review]
Indentation needed

File: .circleci/config.yml (hunk 1 above)
Comment (dprokop) [reply .circleci/config.yml]: Yup, fixed

File: .circleci/config.yml [hunk 2]
```diff
@@ -100,6 +100,10 @@ jobs:
       - attach_workspace:
           at: /tmp/workspace
       - checkout
+      - run:
+          name: Increase node momory
```
Comment (aknuds1) [inline .circleci/config.yml]:
```suggestion
          name: Increase node memory
```

File: .circleci/config.yml [hunk 3]
```diff
@@ -1168,6 +1172,10 @@ jobs:
     working_directory: /go/src/github.com/grafana/grafana
     steps:
       - checkout
+      - run:
+          name: Increase node momory
```
Comment (aknuds1) [inline .circleci/config.yml]:
```suggestion
          name: Increase node memory
```

**Review by aknuds1 — CHANGES_REQUESTED** [review]
Noticed typos :)

**Review by aknuds1 — APPROVED** [review]
LGTM

**Pull request merged**